feature-libs/product-configurator/rulebased/core/state/effects/configurator-basic.effect.ts

```diff
@@ -269,7 +269,8 @@ export class ConfiguratorBasicEffects {
                 map((currentGroupId) => {
                   const groupIdFromPayload =
                     this.configuratorBasicEffectService.getFirstGroupWithAttributes(
-                      payload
+                      payload,
+                      currentGroupId?.startsWith('CONFLICT')
```

Comment:
Please add comment that commerce will always have group ID starting with 'CONFLICT' for group type Configurator.GroupType CONFLICT_GROUP

Reply:
Done